composer.json

```diff
@@ -13,7 +13,7 @@
         "php": "^7.1 || ^8.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9",
+        "phpunit/phpunit": "^10",
```

Comment:
@aashwin-rvvup this should change to `^9 || ^10` because PHPUnit `10.x.x` supports PHP `v8.1` or greater
https://phpunit.de/supported-versions.html